README.md

```diff
@@ -2664,6 +2664,7 @@ _General utilities and tools to make your life easier._
 - [equalizer](https://github.com/reugn/equalizer) - Quota manager and rate limiter collection for Go.
 - [ergo](https://github.com/cristianoliveira/ergo) - The management of multiple local services running over different ports made easy.
 - [evaluator](https://github.com/nullne/evaluator) - Evaluate an expression dynamically based on s-expression. It's simple and easy to extend.
+- [Failsafe-go](https://failsafe-go.dev/) - Fault tolerance and resilience patterns for Go.
```

Comment:
Accordingly to [Contribution Guidelines](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md):
> the link added to `README.md` is the same as the link to the repository mentioned in the body of the PR.

The link should be https://github.com/failsafe-go/failsafe-go, as mentioned in the PR body.